modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/H2QueryInfo.java

```diff
@@ -71,9 +75,10 @@ public class H2QueryInfo {
      * @param sql Query statement.
      * @param node Originator node.
      * @param runningQryId Query id assigned by {@link RunningQueryManager}.
+     * @param label Query label.
      */
     public H2QueryInfo(QueryType type, PreparedStatement stmt, String sql, ClusterNode node,
-        Long runningQryId) {
+        @Nullable Long runningQryId, @Nullable String label) {
```

Comment:
1. why do we need to change nullability for : runningQryId ?
2. i see that in such a case null value can be equated to not nullable field ?
3. label - the same

Reply:
please check
``
    public void printLogMessage(IgniteLogger log, String msg, String additionalInfo) {
        String globalQueryId = runningQryId == null ? "(unknown)" // compatibility with old versions.
            : QueryUtils.globalQueryId(node, runningQryId);
```

AFAIK, this field can be `nul`l in case of some old nodes